app/models/loginAttempts.ts

```diff
@@ -0,0 +1,49 @@
+import type { User } from "@prisma/client";
+import { prisma } from "~/db.server";
+import { updateUser } from "./user.server";
+
+export async function createLoginAttempt(userId: User['id']) {
+  return prisma.loginAttempts.create({ data: { userId, attempts: 1 } })
```

Comment:
upon the first incorrect attempt, we create a row tied to the userId and set attempts to one